packages/mermaid/src/docs/community/development.md

```diff
@@ -1,358 +1,162 @@
+You can install the requirements natively on your machine or in Docker by your choice.
 
-Ex:
+<PlatformSelector :selectedPlatform="selectedPlatform" v-on:change="setPlatform" />
 
-```bash
-# Change into the mermaid directory (the top level director of the mermaid project repository)
-cd mermaid
-# npx is required for first install because volta does not support pnpm yet
-npx pnpm install
-```
+<div v-if="selectedPlatform === 'native'">
 
-#### 3. Verify Everything Is Working
+These are the tools we use for working with the code and documentation:
 
-Once you have installed pnpm, you can run the `test` script to verify that pnpm is working _and_ that the repository has been cloned correctly:
+- [volta](https://volta.sh/) to manage node versions.
+- [Node.js](https://nodejs.org/en/). `volta install node`
+- [pnpm](https://pnpm.io/) package manager. `volta install pnpm`
+- [npx](https://docs.npmjs.com/cli/v8/commands/npx) the packaged executor in npm. This is needed [to install pnpm.](#install-packages)
 
-```bash
-pnpm test
-```
+Follow [the setup steps below](#setup) to install them and start the development
 
-The `test` script and others are in the top-level `package.json` file.
+</div>
 
-All tests should run successfully without any errors or failures. (You might see _lint_ or _formatting_ warnings; those are ok during this step.)
+<div v-if="selectedPlatform === 'docker'">
 
-### Docker
+[Install Docker](https://docs.docker.com/engine/install/). And that is pretty much all you need.
 
-If you are using docker and docker-compose, you have self-documented `run` bash script, which is a convenient alias for docker-compose commands:
+Optionally, to run GUI (Cypress) within Docker you will also need X11 server installed.
+Maybe you already have it installed, so check it first:
 
 ```bash
-./run install # npx pnpm install
-./run test # pnpm test
+echo $DISPLAY
 ```
 
-## Contributing Code
-
-The basic steps for contributing code are:
+If variable `$DISPLAY` is not empty, then it must be working. Otherwise install it.
 
-```mermaid
-graph LR
-    git[1. Checkout a  git branch] --> codeTest[2. Write tests and code] --> doc[3. Update documentation] --> submit[4. Submit a PR] --> review[5. Review and merge]
-```
+</div>
 
-1. **Create** and checkout a git branch and work on your code in the branch
-2. Write and update **tests** (unit and perhaps even integration (e2e) tests) (If you do TDD/BDD, the order might be different.)
-3. **Let users know** that things have changed or been added in the documents! This is often overlooked, but _critical_
-4. **Submit** your code as a _pull request_.
-5. Maintainers will **review** your code. If there are no changes necessary, the PR will be merged. Otherwise, make the requested changes and repeat.
+## Setup and Launch
 
-### 1. Checkout a git branch
+<PlatformSelector :selectedPlatform="selectedPlatform" v-on:change="setPlatform" />
```

Comment:
Can we keep the docs pure markdown?
This is a small feature, but it's a slippery slope, and we might have to do another BIG change to get out of vitepress later.

Reply:
Removed